airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresSqlGenerator.java

```diff
@@ -60,8 +61,28 @@ public class PostgresSqlGenerator extends JdbcSqlGenerator {
       "timestamptz", "timestamp with time zone",
       "timetz", "time with time zone");
 
+  private final NamingConventionTransformer namingTransformer;
```

Comment:
nit: make the superclass field `protected` and then delete this field?

(... and then we'd need to release the cdk for this pr, sorry :( )